core/src/main/java/org/bitcoinj/quorums/ChainLocksHandler.java

```diff
@@ -643,9 +630,21 @@ public void addCoinbaseChainLock(Sha256Hash blockHash, int ancestor, BLSSignatur
         }
     }
 
+    /**
+     * get the coinbase CL for a block
+     * @param blockHash block hash of the block
+     * @return the ChainLockSignature or null if not found
+     */
     public ChainLockSignature getCoinbaseChainLock(Sha256Hash blockHash) {
-        System.out.println("ChainLock Map");
-        coinbaseChainlockMap.values().forEach(System.out::println);
         return coinbaseChainlockMap.get(blockHash);
     }
+
+    /**
+     * get the CL for a block
+     * @param blockHash block hash of the block
+     * @return the ChainLockSignature or null if not found
+     */
+    public ChainLockSignature getChainLock(Sha256Hash blockHash) {
+        return chainlockMap.get(blockHash);
+    }
```

Comment:
Codecov reports this is not covered by tests, but after that report was run `ChainLocksHandlerTest` was added to improve coverage of this class.